lib/fluent/command/cat.rb

```diff
@@ -200,7 +202,8 @@ def write_impl(array)
     end
 
     begin
-      socket.write [@tag, array].to_msgpack
+      packer = Fluent::Engine.msgpack_factory.packer
```

Comment:
It's better to use `Fluent::MessagePackFactory.packer` than using `Fluent::Engine`, because it makes internal dependency problem minimized.